### PR TITLE
[WIP] Removed Mono workarounds

### PR DESCRIPTION
**Pull request by alexvaluyskiy**
- Removed MonoConcurrentQueue

### CODE REVIEW

File: src/core/Akka/Util/MonotonicClock.cs [hunk 1]
```diff
@@ -19,9 +19,6 @@ internal static class MonotonicClock
     {
         private static readonly Stopwatch Stopwatch = Stopwatch.StartNew();
 
-        [DllImport("kernel32")]
-        private static extern ulong GetTickCount64();
```
Comment (Aaronontheweb) [inline src/core/Akka/Util/MonotonicClock.cs]: @alexvaluyskiy why was this removed?

Comment (alexvaluyskiy) [inline src/core/Akka/Util/MonotonicClock.cs]: We have already discussed it few months ago. I can't use it in NetCore, and you recommended to remove it

Comment (Aaronontheweb) [inline src/core/Akka/Util/MonotonicClock.cs]: ah right. That's what I thought.
